www/src/components/shell/onboarding/OnboardingFooter.js

```diff
@@ -0,0 +1,52 @@
+import { A, Flex } from 'honorable'
+import { useNavigate } from 'react-router-dom'
+import { useMutation } from '@apollo/client'
+
+import { UPDATE_USER } from '../../users/queries'
+import { OnboardingStatus } from '../../profile/types'
+
+function OnboardingFooter() {
+  const navigate = useNavigate()
+  const [mutation] = useMutation(UPDATE_USER, {
```

Comment:
It might be worth making this its own hook that can be reused elsewhere

Reply:
Left a comment in a issue on Linear to remember about it.